module/purpose/Goalie/src/Goalie.cpp

```diff
@@ -0,0 +1,102 @@
+#include "Goalie.hpp"
+
+#include "extension/Behaviour.hpp"
+#include "extension/Configuration.hpp"
+
+#include "message/input/GameState.hpp"
+#include "message/planning/LookAround.hpp"
+#include "message/purpose/Goalie.hpp"
+#include "message/strategy/DiveToBall.hpp"
+#include "message/strategy/LookAtFeature.hpp"
+#include "message/strategy/Ready.hpp"
+#include "message/strategy/StandStill.hpp"
+#include "message/strategy/WalkToFieldPosition.hpp"
+
+#include "utility/support/yaml_expression.hpp"
+
+namespace module::purpose {
+    using message::input::GameState;
+    using Phase    = message::input::GameState::Data::Phase;
+    using GameMode = message::input::GameState::Data::Mode;
+    using message::planning::LookAround;
+    using message::strategy::DiveToBall;
+    using message::strategy::LookAtBall;
+    using message::strategy::Ready;
```

Comment:
can be removed, this was the old version of ready before we had walk to field position